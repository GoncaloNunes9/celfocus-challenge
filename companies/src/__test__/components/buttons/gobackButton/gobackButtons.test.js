import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import GoBackButton from '../../../../components/buttons/goBackButton';

describe('Button GoBack', () => {
  it('renders button to go back', () => {
    const wrapper = shallow(<GoBackButton />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})