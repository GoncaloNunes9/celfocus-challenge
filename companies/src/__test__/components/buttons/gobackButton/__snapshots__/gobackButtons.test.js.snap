// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button GoBack renders button to go back 1`] = `
<button
  onClick={[Function]}
>
  Go Back
</button>
`;
